cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinCountry.java

```diff
@@ -0,0 +1,19 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.CODE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.CODE_ALPHA_3;
+import static no.unit.nva.cristin.model.JsonPropertyNames.NAME;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+import no.unit.nva.commons.json.JsonSerializable;
+import no.unit.nva.cristin.person.model.nva.Country;
+
+public record CristinCountry(@JsonProperty(CODE) String code,
```

Comment:
In many ways, I'd prefer an id URI on each country that could be referenced and de-referenced

Reply:
Possibly a new PR

Reply:
do you have an example uri we should use?

Reply:
probably, the current /country/ + unique code (i.e. the ISO-code)

Reply:
iso 2 or 3 letter alpha3? we have both